core/capabilities/ccip/ccipevm/msghasher.go

```diff
@@ -102,15 +126,18 @@ func (h *MessageHasherV1) Hash(_ context.Context, msg cciptypes.Message) (ccipty
 	packedValues, err := h.abiEncode(
 		"encodeFinalHashPreimage",
 		leafDomainSeparator,
-		utils.Keccak256Fixed(metaDataHashInput),
+		utils.Keccak256Fixed(metaDataHashInput), // metaDataHash
 		utils.Keccak256Fixed(fixedSizeFieldsEncoded),
+		utils.Keccak256Fixed(msg.Sender),
 		utils.Keccak256Fixed(msg.Data),
 		utils.Keccak256Fixed(encodedRampTokenAmounts),
```

Comment:
I thought these were all changed to the version in [chainlink-common](https://github.com/smartcontractkit/chainlink-common/blob/main/pkg/hashutil/keccak.go#L30), did we lose that in one of the merges?